dlp/snippets/inspect/inspect_datastore.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inspect
+
+// [START dlp_inspect_datastore]
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/pubsub"
+	dlppb "google.golang.org/genproto/googleapis/privacy/dlp/v2"
+)
+
+// inspectDatastore searches for the given info types in the given dataset kind.
+func inspectDatastore(w io.Writer, projectID string, infoTypes []string, customDictionaries []string, customRegexes []string, pubSubTopic, pubSubSub, dataProject, namespaceID, kind string) error {
+	// projectID := "my-project-id"
+	// infoTypes := []string{"US_SOCIAL_SECURITY_NUMBER"}
+	// customDictionaries := []string{...}
+	// customRegexes := []string{...}
+	// pubSubTopic := "dlp-risk-sample-topic"
+	// pubSubSub := "dlp-risk-sample-sub"
+	// namespaceID := "namespace-id"
+	// kind := "MyKind"
+
+	ctx := context.Background()
+	client, err := dlp.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("dlp.NewClient: %v", err)
+	}
+
+	// Convert the info type strings to a list of InfoTypes.
+	var i []*dlppb.InfoType
```

Comment:
var infoTypes ...

(it's used way way way below)

Reply:
Done. Had to rename the arg to `infoTypeNames`, which is better anyway.